### ISSUE TITLE
imlib2

### ISSUE THREAD

**Issue by 0n0w1c** [clearlinux/distribution#1091]
imlib2 is a needed for openbox, which is the default window manager for LXQt. However, oddly it is not included in the bundle. I do have a request for openbox to be added to the LXQt bundle but has yet to assigned, soo... my attempt to build openbox via source code.

**Comment by 0n0w1c** [comment]
I believe the imlib2 library does exist as a package, however it does not seem to be included in any existing bundles. Please add it to some bundle (or separate) so I can proceed.

**Comment by lebensterben** [comment]
```
❯ sudo swupd search-file -l libImlib2
Downloading Clear Linux manifests (14.06 MB)
        ...100%
Searching for 'libImlib2'

Bundle conky (2 MB to install)
        /usr/lib64/libImlib2.so.1.5.1
        /usr/lib64/libImlib2.so.1

Bundle feh (1 MB to install)
        /usr/lib64/libImlib2.so.1.5.1
        /usr/lib64/libImlib2.so.1

Bundle ssocr (1 MB to install)
        /usr/lib64/libImlib2.so.1.5.1
        /usr/lib64/libImlib2.so.1
```

Maybe install one of those?

**Comment by 0n0w1c** [comment]
There are potentially multiple pieces to the Imlib2 package, as there are "filters" that can be built too, depending on how it is compiled. I did install Imlib2 via the conky bundle. However, the configure script for openbox does not find what it is looking for, from the Imlib2 package. I may need a devpkg-libImlib2 bundle.

**Comment by fenrus75** [comment]
isn't the real answer to this github issue to add openbox to desktop-lxqt?
(if so I just did that)

**Comment by 0n0w1c** [comment]
I opened this request several weeks ago to have openbox added to the LXQt bundle but it does not seem to have gained traction.
[https://github.com/clearlinux/distribution/issues/1002](url)

**Comment by anselmolsm** [comment]
Adding openbox to desktop-lxqt fixes #1002 and is enough for users. A devpkg still makes sense for those who want to build from sources ;)

**Comment by ahkok** [comment]
I believe this can thus be closed.
